### PR TITLE
Add Zone Air Terminal Primary Damper Position Output to AirTerminal:SingleDuct:ParallelPIU:Reheat and AirTerminal:SingleDuct:SeriesPIU:Reheat

### LINKED ISSUE #8065: Missing damper position output for AirTerminal:SingleDuct:ParallelPIU:Reheat
Issue overview
--------------
It would be helpful if damper position was added to the list of outputs for AirTerminal:SingleDuct:ParallelPIU:Reheat as it is for AirTerminal:SingleDuct:VAV:Reheat. Currently you have to compare the actual airflow rate to the max air flow rate to determine the damper % open. 

### Details
Some additional details for this issue (if relevant):
 - EnergyPlus V9.3

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by matthew-larson**
Pull request overview
---------------------
 - Fixes #8065 
 - This pull request adds an output called Zone Air Terminal Primary Damper Position to the list of outputs for AirTerminal:SingleDuct:ParallelPIU:Reheat and AirTerminal:SingleDuct:SeriesPIU:Reheat as it is available for AirTerminal:SingleDuct:VAV:Reheat.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults

### CODE REVIEW

File: tst/EnergyPlus/unit/PoweredInductionUnits.unit.cc [hunk 1]
```diff
@@ -264,36 +268,43 @@ TEST_F(EnergyPlusFixture, ParallelPIUTest1)
...
     DataLoopNode::Node(PriNodeNum).MassFlowRate = PoweredInductionUnits::PIU(SysNum).MaxPriAirMassFlow;
     DataLoopNode::Node(PriNodeNum).MassFlowRateMaxAvail = PoweredInductionUnits::PIU(SysNum).MaxPriAirMassFlow;
+    DataLoopNode::Node(PriNodeNum).MassFlowRateMinAvail = PoweredInductionUnits::PIU(SysNum).MinPriAirMassFlow;
     DataZoneEnergyDemands::ZoneSysEnergyDemand(1).RemainingOutputRequired = 2000.0; // Heating load - expect min flow rate
     DataZoneEnergyDemands::CurDeadBandOrSetback(1) = true;
     DataHVACGlobals::TurnFansOn = true;
     DataHVACGlobals::TurnZoneFansOnlyOn = false;
     PoweredInductionUnits::CalcParallelPIU(state, SysNum, ZoneNum, ZoneNodeNum, FirstHVACIteration);
-    EXPECT_EQ(0.0, DataLoopNode::Node(SecNodeNum).MassFlowRate);
+    EXPECT_EQ(SecMaxMassFlow, DataLoopNode::Node(SecNodeNum).MassFlowRate);
```
Comment (matthew-larson) [inline tst/EnergyPlus/unit/PoweredInductionUnits.unit.cc:282]: Adding the minimum primary flow results in reheat being required and calls for secondary flow, similar to the SeriesPIU test.

File: tst/EnergyPlus/unit/PoweredInductionUnits.unit.cc [hunk 2]
```diff
@@ -264,36 +268,43 @@ TEST_F(EnergyPlusFixture, ParallelPIUTest1)
...
     DataLoopNode::Node(PriNodeNum).MassFlowRateMaxAvail = PoweredInductionUnits::PIU(SysNum).MaxPriAirMassFlow;
-    DataZoneEnergyDemands::ZoneSysEnergyDemand(1).RemainingOutputRequired = -2000.0; // Heating load - expect min flow rate
+    DataLoopNode::Node(PriNodeNum).MassFlowRateMinAvail = PoweredInductionUnits::PIU(SysNum).MinPriAirMassFlow;
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(1).RemainingOutputRequired = -2000.0; // Cooling load - expect min flow rate
     DataZoneEnergyDemands::CurDeadBandOrSetback(1) = false;
     DataHVACGlobals::TurnFansOn = true;
     DataHVACGlobals::TurnZoneFansOnlyOn = false;
     PoweredInductionUnits::CalcParallelPIU(state, SysNum, ZoneNum, ZoneNodeNum, FirstHVACIteration);
-    EXPECT_EQ(0.0, DataLoopNode::Node(SecNodeNum).MassFlowRate);
+    EXPECT_EQ(SecMaxMassFlow, DataLoopNode::Node(SecNodeNum).MassFlowRate);
```
Comment (matthew-larson) [inline tst/EnergyPlus/unit/PoweredInductionUnits.unit.cc:306]: Same as above, even though the zone has a cooling load, the minimum primary air is overcooling, thus requiring reheat.  Probably should be corrected.

**Comment by matthew-larson** [comment]
All diffs are due to the addition of the output variable to the test files and the 1 failed test is unrelated to this change.

**Review by mitchute — APPROVED** [review]
These changes look good. Tests all pass locally and I ran 5ZoneAirCooledConvCoef to make sure the expected results come out. This is ready to go.

**Pull request merged**